app/views/hunters/show.html.erb

```diff
@@ -1,6 +1,6 @@
 <div class="columns">
   <div class="column">
-    <h1><%= @hunter.name %>, <%= @hunter.playbook.name %></h1>
+    <h1><%= @hunter.name %>, <%= link_to @hunter.playbook.name, playbook_path(@hunter.playbook) %></h1>
```

Comment:
Could we use the `link_to_playbook` method from the playbooks_helper here?

https://github.com/ChaelCodes/HuntersKeepers/blob/64e471a7b5d37063b2a2195944fd6ccbabbceff2/app/helpers/playbooks_helper.rb#L5-L7

Reply:
Thank you!